massdriver-package-alarm/providers.tf

```
@@ -0,0 +1,7 @@
+terraform {
+  required_providers {
+    massdriver = {
+      source = "massdriver-cloud/massdriver"
```

I think here we should set the minimum version of the massdriver provider to the version we need for this module.

```suggestion
      source = "massdriver-cloud/massdriver"
      version = ">= 1.1.5"
```

I did this for [mdxc](https://github.com/massdriver-cloud/terraform-modules/blob/main/massdriver-application-azure-app-service/_providers.tf#L17) and it worked well. It's nice because if anyone is running things locally, this will force a new version to be downloaded from the registry. It also ensures that this module will work independently.

---

Good suggestion